less/2010/eclipsecon/images/eclipse-foundation-grey-orange.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this can be removed since it was there for the footer and we are now getting that image directly in the static folder of the website.